styles.css

```diff
@@ -0,0 +1,54 @@
+* {
+    box-sizing: border-box;
+}
+
+
+
+.collection_item {
+
```

Comment:
these are also extra newlines, and they're inconsistently applied in this file, so clean these up too please